wandb/sync/sync.py

```diff
@@ -201,6 +217,21 @@ def _send_tensorboard(self, tb_root, tb_logdirs, send_manager):
         handle_manager.finish()
         send_manager.finish()
 
+    def _robust_scan(self, ds):
+        """Attempt to scan data, handling incomplete files"""
+        try:
+            return ds.scan_data()
+        except AssertionError as e:
+            if ds.in_last_block():
```

Comment:
This is the fix for in progress syncing.  If we get an assertion error and are in the last block of the leveldb file we just print a warning but don't raise an exception.